platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -100,6 +108,7 @@ public void init() throws Exception {
             Files.createDirectories(exportDirectory);
             configurationAdminMigration.export(exportDirectory);
             migrationWarnings.addAll(systemConfigurationMigration.export(exportDirectory));
```

Comment:
We are going to remove the line above after we get all of the Migratables finished..right?